asv.conf.json

```diff
@@ -1,10 +1,10 @@
 {
-    "version": 1,
+    "version": 2,
```

Comment:
I think this should stay as `1`. 

See [logs](https://github.com/django/django-asv/pull/72/checks#step:6:10).

Reply:
Reverted :+1: